gcc/rust/expand/rust-macro-expand.h

```diff
@@ -57,11 +57,21 @@ struct MatchedFragment
   size_t match_amount;
 
   MatchedFragment (std::string identifier, size_t token_offset_begin,
-		   size_t token_offset_end, size_t match_amount = 0)
+		   size_t token_offset_end, size_t match_amount = 1)
```

Comment:
is this change meant to be here?

Reply:
Yes! The default behavior should be that there is a single match when we create it. It's only when we want to specifically create an empty match that we set it to zero. The previous implementation was slightly flawed